api/v1/openapi.yaml

```diff
@@ -3506,6 +3506,15 @@ definitions:
         type: array
         items:
           "$ref": "#/definitions/BgpPeerFamilies"
+      connect-retry-time-seconds:
+        description: Initial value for the BGP ConnectRetryTimer (RFC 4271, Section 8) in seconds
```

Comment:
What does "Initial value" means? I feel a bit odd since it can be updated. Same for other parameters.

Reply:
It is based on wording from the RFC 4271:

> ConnectRetryTime is a mandatory FSM attribute that stores the initial
   value for the ConnectRetryTimer.  The suggested default value for the
   ConnectRetryTime is 120 seconds.

or:
> the local system starts the ConnectRetryTimer with the initial value

Basically the in sense of RFC the value of `ConnectRetryTimer` (and others similarly) is originally initiated with the `initial value` and then being decremented each second.

But I agree that it might be misleading, maybe we can change it to something like `BGP ConnectRetryTimer interval`?

Reply:
Ah, no. I was just reading the RFC and understood the meaning. I think it's fine :)